contributing_to_docs/contributing.adoc

```diff
@@ -164,12 +165,18 @@ file to determine which topic files to include. Therefore, all new assemblies th
 are created must be included in the `&#95;topic&#95;map.yml` file in
 order to be processed by the build system.
 
+For all supported versions, the `&#95;topic&#95;map.yml` files are available in the `&#95;topic&#95;maps` folder. Older versions support `&#95;topic&#95;map.yml` file in the root folder.
```

Comment:
You could change "the `_topic_map.yml` files are available" to "the topic map files are available", since some of them have different names.